whylog/tests/test_constraints/__init__.py

```diff
@@ -0,0 +1,23 @@
+from unittest import TestCase
+
+from whylog.constraints import IdenticalConstraint
+from whylog.constraints.exceptions import ConstraintVerificationError
+
+
+class TestIdenticalConstraint(TestCase):
+    def test_get_param_names(self):
+        assert IdenticalConstraint.get_param_names() == []
+
+    def test_get_group_count(self):
+        assert IdenticalConstraint.get_groups_count() == (2, None)
+
+    def test_verify_success(self):
+        IdenticalConstraint.verify({}, ['comp1', 'comp1', 'comp1'])
+
+    def test_verify_fail(self):
+        self.assertRaises(
+            ConstraintVerificationError,
+            IdenticalConstraint.verify,
+            {},
+            ['comp1', 'hello', 'comp1'],
```

Comment:
This assert is applied to function which is not called. Don't separate functions params with comma but call this function with this params.